dc/core/MasterToWallChannel.h

```diff
@@ -76,19 +76,19 @@ public slots:
      * Send the given DisplayGroup to the wall processes.
      * @param displayGroup The DisplayGroup to send
      */
-    void sendAsync( DisplayGroupPtr displayGroup );
+    void sendDisplayGroup( DisplayGroupPtr displayGroup );
```

Comment:
The send(Frame) documentation is correct (and also applies to sendQuit()). They behave as one would expect: Qt automatically set up a queued connection and serializes the frame in the Channel thread. This couldn't be done for the other 3 methods as stated above, because the objects may be modified concurrently by the main thread, and we resolved on providing these special sendAsync functions, which have to be used in a special and documented way. As long as there is no confusion possible for the user, I am not fixed on specific names, but at least the very least documentation must be adapted.